src/main.py

```diff
@@ -13,8 +13,11 @@ class Main:
     """
     Driver class
     """
-
+    
     def __init__(self) -> None:
+        print("Enter the name of the csv file")
+        csv_file_path = input()
```

Comment:
Would be preferable to get the path from the command line using the argparse module.

https://docs.python.org/3/library/argparse.html

This parser should be in the root of the project.